Sources/Mantis/Mantis.swift

```diff
@@ -91,7 +91,7 @@ public struct CropToolbarConfig {
     public var optionButtonFontSizeForPad: CGFloat = 20
     public var cropToolbarHeightForVertialOrientation: CGFloat = 44
     public var cropToolbarWidthForHorizontalOrientation: CGFloat = 80
-    
+    public var useRatioPresenter = true
```

Comment:
For me, I would like to use a enum type like 
```swift
public enum RatioCandidatesShowType {
    case presentRatioList
    case alwaysShowRatioList
}
```

Then add a `ratioCandidatesShowType: RatioCandidatesShowType ` to CropToolbarConfig with the intialValue  . presentRatioList